readme.md

```diff
@@ -1,143 +1,152 @@
-# openwisp-wifi-login-pages
```

Comment:
can you avoid changing all these lines in the readme please?

Reply:
done reverted back